### PR TITLE
Preparations for "featured" posts

### PR DESCRIPTION
**Pull request by annando**
Incoming activities for featured posts are now stored and can be retrieved.

In a next PR these featured posts can then be displayed accordingly. Also in a further step we will transmit this information as well.

### CODE REVIEW

**Comment by tobiasd** [comment]
And _what is_ a featured post?

**Comment by MrPetovan** [comment]
Is it a pinned post?

File: database.sql [hunk 1]
```diff
@@ -1062,6 +1062,17 @@ CREATE TABLE IF NOT EXISTS `post-category` (
 	FOREIGN KEY (`tid`) REFERENCES `tag` (`id`) ON UPDATE RESTRICT ON DELETE RESTRICT
 ) DEFAULT COLLATE utf8mb4_general_ci COMMENT='post relation to categories';
 
+--
+-- TABLE post-collection
+--
+CREATE TABLE IF NOT EXISTS `post-collection` (
+	`uri-id` int unsigned NOT NULL COMMENT 'Id of the item-uri table entry that contains the item uri',
+	`type` tinyint unsigned NOT NULL DEFAULT 0 COMMENT '',
```
Comment (MrPetovan) [inline database.sql]: Please add the possible values in the comment

**Review by MrPetovan — DISMISSED** [review]

**Comment by annando** [comment]
@tobiasd a featured post is the Mastodon name for a pinned post.

File: database.sql (hunk 1 above)
Comment (annando) [reply database.sql]: Done

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
Code standards test is failing

**Comment by annando** [comment]
@MrPetovan not anymore 😁

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**